app/javascript/modules/listingDetailsAside/ListingDetailsWaitlist.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react"
+import { QuantityRowSection, t } from "@bloom-housing/ui-components"
+import { RailsListing } from "../listings/SharedHelpers"
+import { isLotteryComplete, isOpen } from "../../util/listingUtil"
+
+export interface ListingDetailsWaitlistProps {
+  listing: RailsListing
+}
+
+export const ListingDetailsWaitlist = ({ listing }: ListingDetailsWaitlistProps) => {
+  if (!listing.hasWaitlist || (!isOpen(listing) && !isLotteryComplete(listing))) return null
```

Comment:
any preference or thoughts on returning null vs <></>?

Reply:
@kramduckner good question. the react docs recommend returning null https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering